src/site/content/en/patterns/web-vitals-patterns/images/img-tag/assets/body.html

```diff
@@ -0,0 +1 @@
+<img src="chicken.jpg" width="267" height="400">
```

Comment:
If you add style.css file with multiple classes for each version of object-fit: xxx property, they will also get displayed in the samples tabs. Or you can add n x img tag here, each time with different style="..." attribute.